measurecrack.py

```diff
@@ -0,0 +1,175 @@
+import cv2, imutils
+import numpy as np
+
+# Measures crack using raio method
+def measure_crack(image):
+    #Get dimensions of blue rectangle
+    width, height = blue_rectangle(image)
+    
+    # Return larger dimension. Multiply ratio by 1.85
+    if height > width:
+        return (height/width)*1.9
+    else:
+        return (width/height)*1.9
+
+# Finds dimensions of a blue rectangle    	
+def blue_rectangle(image):
+    # Convert BGR to HSV
+    hsv = cv2.cvtColor(image, cv2.COLOR_BGR2HSV)
+
+    # define range of blue color in HSV
+    lower_blue = np.array([90,50,50])
+    upper_blue = np.array([130,255,255])
+
+    # Threshold the HSV image to get only blue colors
+    mask = cv2.inRange(hsv, lower_blue, upper_blue)
+    
+    # Find contours
+    contours, hierarchy = cv2.findContours(mask.copy(), cv2.RETR_EXTERNAL, cv2.CHAIN_APPROX_SIMPLE)
+    img = cv2.drawContours(mask, contours, -1, (50,255,0), 3)
+    
+    # Find max area contour
+    maxArea = 0
+    
+    for c in contours:
+        if cv2.contourArea(c) > maxArea:
+            maxArea = cv2.contourArea(c)
+            maxc = c
+    
+    # Find minimum bounding rectangle
+    rect = cv2.minAreaRect(maxc)
+    width = rect[1][0]
+    height = rect[1][1]
+    return width, height
+
+# Measures crack using perspective transformation method
+def measure_crack2(image):
+    # define range of color in HSV
```

Comment:
i feel like the values youres putting into these variables should probably have less magic numbers. 

i shouldnt be asking "why are you choosing this range?"

Reply:
yeah, you should add a comment about why this range was chosen or give the variables/numbers a more descriptive name